packages/ui-shared/src/components/TextWithIcon/TextWithIcon.tsx

```diff
@@ -16,25 +16,50 @@ export interface ITextWithIconProps {
     };
     /** Дополнительный класс для корневого элемента */
     className?: string;
+    /** Отключить ограничение ширины */
+    isFullWidth?: boolean;
+    /** Выравнивание по центру на мобильных (будет удалено в следующей мажорной версии) */
```

Comment:
достаточно указать `/* @deprecated */` перед свойством

```suggestion
    /* @deprecated */
    /** Выравнивание по центру на мобильных */
```